metadata-ingestion-modules/airflow-plugin/tox.ini

```diff
@@ -14,7 +14,11 @@ deps =
     # Airflow version
     airflow21: apache-airflow~=2.1.0
     airflow22: apache-airflow~=2.2.0
-    airflow24: apache-airflow~=2.4.0
+    # On Airflow 2.4 and 2.5, Airflow's constraints file pins pluggy to 1.0.0,
+    # which has caused issues for us before. As such, we now pin it explicitly
+    # to prevent regressions.
+    # See https://github.com/datahub-project/datahub/pull/9365
+    airflow24: apache-airflow~=2.4.0,pluggy==1.0.0
```

Comment:
Why don't you explicitly pin the `pluggy` to `1.0.0` for airflow25 like you did for airflow 24?

I mean like this -> `airflow25: apache-airflow~=2.5.0,pluggy==1.0.0`.